.github/ISSUE_TEMPLATE/cm---issue-template.md

```diff
@@ -0,0 +1,30 @@
+---
+name: CM - Issue template
```

Comment:
Hi @OlaShabalina ,
Can we rename the template file from `cm---issue-template.md` to something like `bug-report.md` ?